en/docs/develop/integration-artifacts/file/ftp-sftp.md

```
@@ -156,445 +195,370 @@ service on ftpsListener {
+4. Choose **Certificate Based Authentication** under **authentication method**. This reveals the **Private Key** and **Username** fields.
+
+5. Enter the **Private Key** record. Click **Record** on the field and supply:
+
+   ```
+   {path: "/path/to/private_key"}
+   ```
+
+   If the private key is passphrase-protected, include the passphrase
+   in the record:
+
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)